srcpkgs/spago/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'spago'
+pkgname=spago
+version=0.16.0
+revision=1
+hostmakedepends="
+	curl
+	ncurses-libtinfo-libs
+	stack
+	tar
+	xz
+	zlib
+	zlib-devel
+"
+short_desc="PureScript package manager & build tool powered by Dhall & package-sets"
+maintainer="Wayne Van Son <waynevanson@gmail.com>"
+license="BSD-3-Clause"
+homepage="https://github.com/purescript/spago"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum="3172cb0a5d5f603d77b0b2b6b22652c202417281340743e6478ceef1055d03c9"
+
+# libtinfo.so.5 patch
+do_build() {
```

Comment:
This function isn't necessary if the patch is handled properly and you use the right build style.